panther_docking/src/dock_pose_publisher_node.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2024 Husarion Sp. z o.o.
```

Comment:
I can guess, how to use it but I cannot find any place in the code where this node should be runned.

Reply:
https://github.com/husarion/panther_ros/pull/431/files#diff-191c7d626c87b7cb35fbad246605eb38bcf4f84839fd3c80bcb93ad4c30be3cdR94